Userland/Libraries/LibWeb/DOM/Node.cpp

```diff
@@ -1736,6 +1736,84 @@ Optional<String> Node::lookup_namespace_uri(Optional<String> prefix) const
     return locate_a_namespace(prefix);
 }
 
+// https://dom.spec.whatwg.org/#locate-a-namespace-prefix
+Optional<String> Node::locate_a_namespace_prefix(Optional<String> const& namespace_) const
```

Comment:
Would you ever expect this to be called on something that isn't an element? If not, would it be better to move this method to `DOM::Element` instead?

Reply:
Yeah, I don't (as evident in the guard clause below). I'll move it to `DOM::Element` since it should make more sense there. Thanks!